src/type.zig

```diff
@@ -5,6 +5,7 @@ const Allocator = std.mem.Allocator;
 const Target = std.Target;
 const Module = @import("Module.zig");
 const log = std.log.scoped(.Type);
+const Air = @import("Air.zig");
```

Comment:
sus

Reply:
Oh good catch, I added this thinking mistakenly that I needed to use instToRef, and then forgot to delete it.